recipe/patches/0001-Do-not-install-documentation.patch

```diff
@@ -0,0 +1,25 @@
+From 5596617baacbe99f0c4dc6d97417d16b6b1b69d8 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Julian=20R=C3=BCth?= <julian.rueth@fsfe.org>
+Date: Thu, 6 Apr 2023 22:14:48 +0200
+Subject: [PATCH] Do not install documentation
```

Comment:
How about including in the "commit message" of the patch an explanation *why* it is used?